### DISCUSSION TITLE
Add short help texts in menu

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#69] [Ideas]
Some menu items have different texts after selecting it, for example menu text `Set level offset` (24) gets `RF Level offset` (14) or `Language` (39) gets `Choose Language` (40).

Background: After a user asked us the question how to connect to **Stationlist**, I realized that is not self-explaining. 

Idea: Additional short (localized) help texts (like `Please enter the IP-address of your PC`) in the menu could avoid such issues.

**Comment by PE5PVB** [comment]
I think it's better to just have a complete manual in wiki. Otherwise you will fill up the screen with this and this uses a lot of extra memory. Maybe this is something you can join?
